CMakeLists.txt

```diff
@@ -486,16 +486,15 @@ if (LLAMA_VULKAN)
 endif()
 
 if (LLAMA_HIPBLAS)
-    list(APPEND CMAKE_PREFIX_PATH /opt/rocm)
-
-    if (NOT ${CMAKE_C_COMPILER_ID} MATCHES "Clang")
-        message(WARNING "Only LLVM is supported for HIP, hint: CC=/opt/rocm/llvm/bin/clang")
-    endif()
-
-    if (NOT ${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
-        message(WARNING "Only LLVM is supported for HIP, hint: CXX=/opt/rocm/llvm/bin/clang++")
+    if ($ENV{ROCM_PATH})
+        set(ROCM_PATH $ENV{ROCM_PAT})
```

Comment:
There's a typo in `$ENV{ROCM_PATH}` here.

Reply:
Fixed, thank you!